doc/source/examples/multiphysics/melting-cavity/melting-cavity.rst

```diff
@@ -0,0 +1,235 @@
+==========================
+Melting Cavity
+==========================
+
+This example simulates a `two-dimensional gallium melting cavity`_. 
+
+.. _two-dimensional gallium melting cavity: https://asmedigitalcollection.asme.org/heattransfer/article/108/1/174/414516/Melting-and-Solidification-of-a-Pure-Metal-on-a
+
+
+----------------------------------
+Features
+----------------------------------
+- Solver: ``gls_navier_stokes_2d`` 
+- Phase change (solid-liquid)
+- Buoyant force (natural convection)
+- Unsteady problem handled by an adaptive BDF2 time-stepping scheme 
+- The use of a python script for post-processing data
+
+
+------------------------
+Location of the example
+------------------------
+``examples/multiphysics/melting_cavity/melting_cavity.prm``
+
+
+-----------------------------
+Description of the case
+-----------------------------
+
+The melting of metals (gallium in this example) with natural convection within a cavity is a well-known benchmark. The following schematic describes the geometry and dimensions of the simulation in the :math:`(x,y)` plane:
```

Comment:
I think it would be good to add somewhere in this section the final equations we solving. This way everybody would be clear on what the buoyancy term is :).